provider/aws.go

```diff
@@ -254,6 +257,39 @@ func (p *AWSProvider) submitChanges(changes []*route53.Change) error {
 	return nil
 }
 
+func limitChangeSet(cs []*route53.Change) []*route53.Change {
+	limit := 4000
```

Comment:
this should be a constant defined on the top of the file

Reply:
done